frontend/packages/console-dynamic-plugin-sdk/README.md

```
@@ -516,22 +516,16 @@ Before publishing, it's recommended to log into your npm user account:
 npm login
 ```
 
-Build all distributable [SDK packages](#sdk-packages) into `dist` directory:
+Build all distributable [SDK packages](#sdk-packages) into `dist` directory and set their version:
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)